addons/mail/tests/test_mail_template.py

```diff
@@ -443,7 +443,7 @@ class TestSearchTemplateCategory(MailCommon):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.mail_template = cls.env['mail.template'].with_context(active_test=False)
+        cls.mail_template = cls.env['mail.template'].with_context(active_test=False).with_user(cls.user_employee)
```

Comment:
Actually tests should run using real users (use @users decorator for example). It was a mistake doing everything as root indeed.

Reply:
@tde-banana-odoo The new version is refactoring the test to run under a `@users('employee')` decorator, but the diff is a bit larger.